video_xblock/static/css/videojs.css

```diff
@@ -455,6 +455,11 @@ body {
     border-left: 2px solid #cb598d;
 }
 
+.vjs-playback-rate .vjs-menu .vjs-menu-content li.selected-item {
```

Comment:
Why didn't you extend the previous rule instead of creating a new one?

Reply:
Done